test/test-client.js

```diff
@@ -290,4 +318,76 @@ describe('Imgix client:', function describeSuite() {
       assert.equal(expectation, result);
     });
   });
+
+  describe('Sharding', function describeSuite() {
+    describe('CRC', function describeSuite() {
+      it('path resolves to same domain', function testSpec() {
+        var domains = ['my-host1.imgix.net', 'my-host2.imgix.net']
+        var client = new ImgixClient({
+          domains: domains,
+          shard_strategy: ImgixClient.SHARD_STRATEGY_CRC
+        });
+        assert.ok(client.buildURL('/users/1.png').match(domains[0]));
+        assert.ok(client.buildURL('/users/2.png').match(domains[0]));
+        assert.ok(client.buildURL('/users/a.png').match(domains[1]));
+        assert.ok(client.buildURL('/users/1.png').match(domains[0]));
+        assert.ok(client.buildURL('/users/a.png').match(domains[1]));
+        assert.ok(client.buildURL('/users/2.png').match(domains[0]));
+        assert.ok(client.buildURL('/users/b.png').match(domains[1]));
+        assert.ok(client.buildURL('/users/a.png').match(domains[1]));
+      });
+
+      it('single domain sharding', function testSpec() {
+        var domains = 'my-host1.imgix.net';
+        var client = new ImgixClient({
+          domains: domains,
+          shard_strategy: ImgixClient.SHARD_STRATEGY_CRC
+        });
+        assert.ok(client.buildURL('/users/1.png').match(domains[0]));
```

Comment:
I don't think these tests are actually doing the thing you want them to. Since `domains` is a string here and not an array, this is just checking whether the first character of the string appears in the resultant URL.

As a sidenote, if we could rename this variable to `domain` in this test, that would be good.

Reply:
Makes sense. Will update it.